syntax = "proto3";
package rpc;

message rpcMeta{
  string service_name=1;
  string method_name=2;
  int64 data_size=3;
}